src/name_bazaar/ui/events/registrar_events.cljs

```diff
@@ -51,6 +51,42 @@
                    :on-tx-receipt [:district0x.snackbar/show-message
                                    (gstring/format "%s was successfully registered" ens-record-name)]}]})))
 
+(reg-event-fx
+  :registrar/start-auction
+  [interceptors (validate-first-arg (s/keys :req [:ens.record/label]))]
+  (fn [{:keys [:db]} [form-data]]
+    (let [label (normalize (:ens.record/label form-data))
+          ens-record-name (str label constants/registrar-root)]
+      {:dispatch [:district0x/make-transaction
+                  {:name (gstring/format "Start auction for %s" ens-record-name)
+                   :contract-key :registrar
+                   :contract-method :start-auction
+                   :form-data (assoc form-data :ens.record/label-hash (sha3 label))
+                   :result-href (path-for :route.ens-record/detail {:ens.record/name ens-record-name})
+                   :args-order [:ens.record/label-hash]
+                   :tx-opts {:gas 700000 :gas-price default-gas-price}
+                   :on-tx-receipt [:district0x.snackbar/show-message
+                                   (gstring/format "Auction for %s successfully started"
+                                                   ens-record-name)]}]})))
+
+(reg-event-fx
+  :registrar/start-auctions-and-bid
+  [interceptors (validate-first-arg (s/keys :req [:ens.record/label :ens.record/sealed-bid]))]
+  (fn [{:keys [:db]} [form-data]]
+    (let [label (normalize (:ens.record/label form-data))
+          ens-record-name (str label constants/registrar-root)]
+      {:dispatch [:district0x/make-transaction
+                  {:name (gstring/format "Start auction and bid for %s" ens-record-name)
+                   :contract-key :registrar
+                   :contract-method :start-auctions-and-bid
+                   :form-data (assoc form-data :ens.record/label-hashes [(sha3 label)])
+                   :result-href (path-for :route.ens-record/detail {:ens.record/name ens-record-name})
+                   :args-order [:ens.record/label-hashes :ens.record/sealed-bid]
+                   :tx-opts {:gas 700000 :gas-price default-gas-price}
```

Comment:
You'll need `:form-id` here also, to associate form params with this transaction, so loading state can be displayed in form. In this case it can be: 
`:form-id {:ens.record/label label :ens.record/owner owner}`. 
Then you have to create sub `:registrar.start-auction/tx-pending?` with those params